man/man8/zinject.8

```diff
@@ -19,11 +19,11 @@
 .\" CDDL HEADER END
 .\"
 .\" Copyright 2013 Darik Horn <dajhorn@vanadac.com>. All rights reserved.
-.\" Copyright (c) 2024, Klara Inc.
+.\" Copyright (c) 2024, 2025, Klaram Inc.
```

Comment:
A typom :)

Reply:
Unless I can get Allan to rename the company...